src/components/layout/Pagination.js

```diff
@@ -1,21 +1,10 @@
 import React from "react";
-import { Link } from "react-router-dom";
-import styled from "styled-components";
 
-//@todo move completely to styled components (Pagination Nav, is-centered).
-
-const Wrapper = styled.nav`
-  // padding: 10px;
-  // margin-top: 30px;
-  margin-top: 50px;
-  width: 100%;
-`;
-
-const List = styled.ul``;
+import Page from "components/styles/Page";
```

Comment:
it would have been great if this was called Pagination instead of Page, but it would have broken existing Pagination, as that would have needed to be renamed PaginationComponent, or something similar